.chloggen/remove_metrics_proxy.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```

Comment:
Does this change affect the end user? Do end users need to update the metric endpoint?
I _think_ it's an internal change and transparent to the user, so I'd mark it as enhancement.

Reply:
It doesn't affect. it should be transparent